etcdctl/READMEv2.md

```diff
@@ -305,17 +302,15 @@ etcdctl --endpoint http://10.0.28.1:4002,http://10.0.28.2:4002,http://10.0.28.3:
 
 ## Username and Password
 
-If your etcd cluster is protected by [authentication][authentication], you can specify username and password using the [`--username`][username-flag] or `ETCDCTL_USERNAME` environment variable. When `--username` flag or `ETCDCTL_USERNAME` environment variable doesn't contain password, etcdctl will prompt password in interactive mode.
+If the etcd cluster is protected by [authentication][authentication], specify username and password using the [`--username`][username-flag] or `ETCDCTL_USERNAME` environment variable. When `--username` flag or `ETCDCTL_USERNAME` environment variable doesn't contain password, etcdctl will prompt password in interactive mode.
 
 ```sh
 ETCDCTL_USERNAME="root:password" etcdctl set my-key to-a-value
 ```
 
 ## DNS Discovery
 
-If you want to discover your etcd cluster through domain SRV records you can specify
-a `--discovery-srv` flag or `ETCDCTL_DISCOVERY_SRV` environment variable. This option takes
-precedence over the `--endpoint` flag.
+To discover the etcd cluster through domain SRV records,  specify a `--discovery-srv` flag or `ETCDCTL_DISCOVERY_SRV` environment variable. This option takes precedence over the `--endpoint` flag.
```

Comment:
the etcdctl v2 says to use `--endpoints` since `--endpoint` is deprecated.

```
--endpoint value                 DEPRECATED - "--endpoints" should be used instead
--endpoints value                a comma-delimited list of machine addresses in the cluster (default: "http://127.0.0.1:2379,http://127.0.0.1:4001")
```